ehcache-impl/src/main/java/org/ehcache/impl/serialization/CompactJavaSerializer.java

```diff
@@ -80,8 +85,12 @@ public static <T> Class<? extends Serializer<T>> asTypedSerializer() {
 
   @Override
   public void init(final StateRepository stateRepository) {
-    this.persistentState = stateRepository.getPersistentStateHolder("CompactJavaSerializer-ObjectStreamClassIndex", Integer.class, ObjectStreamClass.class, c -> true, null);
-    refreshMappingsFromStateRepository();
+    try {
+      this.persistentState = stateRepository.getPersistentStateHolder("CompactJavaSerializer-ObjectStreamClassIndex", Integer.class, ObjectStreamClass.class, c -> true, null);
+      refreshMappingsFromStateRepository();
+    }catch (CachePersistenceException e){
```

Comment:
Incorrect formatting.

Reply:
The exception is re-thrown as a RuntimeException in FileBasedStateRepository:getPersistentStateHolder(), hence this change will be reverted and formatting will be corrected.